physx-sys/src/lib.rs

```diff
@@ -171,6 +171,48 @@ pub const fn version(major: u32, minor: u32, patch: u32) -> u32 {
 pub type CollisionCallback =
     unsafe extern "C" fn(*mut c_void, *const PxContactPairHeader, *const PxContactPair, count: u32);
 
+pub type TriggerCallback = unsafe extern "C" fn(*mut c_void, *const PxTriggerPair, count: u32);
+
+pub type ConstraintBreakCallback =
+    unsafe extern "C" fn(*mut c_void, *const PxConstraintInfo, count: u32);
+
+pub type WakeSleepCallback =
+    unsafe extern "C" fn(*mut c_void, *const *const PxActor, count: u32, bool);
+
+pub type AdvanceCallback =
+    unsafe extern "C" fn(*mut c_void, *const *const PxRigidBody, *const PxTransform, count: u32);
```

Comment:
Why is only the count named above? Is it required to be? It looks odd.

Reply:
No, it's not. Just cargo culting myself. Removed it.